src/transformers/feature_extraction_sequence_utils.py

```diff
@@ -316,7 +317,9 @@ def _truncate(
         truncation: Optional[bool] = None,
     ):
         """
-        Pad inputs (on left/right and up to predefined length or max length in the batch)
+        Truncate inputs (on left/right and up to predefined length or max length in the batch)
```

Comment:
Can we remove this for now from the feature extraction sequence utils? :-) It'll cause problems with the normalization at the moment, so I'd prefer to not added

Reply:
This PR helped me found some redundant code :-) https://github.com/huggingface/transformers/pull/13051

Reply:
Okey thinking a bit more about it, it should actually be fine to add this feature if the comments here: https://github.com/huggingface/transformers/pull/12913/files#r685115702 are applied & tests are added to https://github.com/huggingface/transformers/blob/3ff2cde5ca4a2d3c622b827d9edf7e3d0b7f4fb7/tests/test_sequence_feature_extraction_common.py#L361 . It could be as simple as just adding a generic parameter `truncation_side` to the test function: https://github.com/huggingface/transformers/blob/3ff2cde5ca4a2d3c622b827d9edf7e3d0b7f4fb7/tests/test_sequence_feature_extraction_common.py#L221 which is then tested for both left and right